libraries/aws_conn.rb

```diff
@@ -28,4 +28,12 @@ def iam_resource
   def iam_client
     @iam_client ||= Aws::IAM::Client.new
   end
+
+  def sts_client
+    @sts_client ||= Aws::STS::Client.new
+  end
+
+  def account_id
+    sts_client.get_caller_identity[:account]
```

Comment:
Not sure if this should belong here, but it did feel like the right place. The account ID is tied to the access key pair used to run the tests, which is what is required for the AWS connection. For reference of this API see http://docs.aws.amazon.com/STS/latest/APIReference/API_GetCallerIdentity.html.